Replicator/tests/ReplicatorAPITest.hh

```diff
@@ -434,6 +435,20 @@ public:
     }
 
 
+    bool requireSG3() {
+        // `_serverName` is set by sendRemoteRequest
+        if (!_serverName)
+            sendRemoteRequest("HEAD", "/");
+        REQUIRE(_serverName.hasPrefix("Couchbase Sync Gateway/"));
+        if (_serverName >= "Couchbase Sync Gateway/3") {
```

Comment:
Good until "/9".

Reply:
Yup, we can update this line then. For now I don't see a need to do more complex parsing.

Reply:
Agreed